Images_And_Graphics_style.css_v1

```diff
@@ -17,29 +17,36 @@ h1 {
 
 h2 {
   color: #000000;
-  font-size: 30px;
+  font-size: 20px;

```

Comment:
I changed the font size of the heading 2 from 30px to 20px because the fot size was to big.